.github/workflows/test-fortran.yml

```diff
@@ -0,0 +1,37 @@
+name: fortran
+on:
+  push:
+    branches:
+      - latest
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  test:
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            compiler: gcc
+            version: 12
+          - os: ubuntu-latest
+            compiler: gcc
+            version: 13
+          - os: macos-13
+            compiler: gcc
+            version: 11
```

Comment:
Is it necessary to test this mix of compilers? Why 11 on Mac but 12 and 13 on linux?